Documentation/building/linux-instructions.md

```diff
@@ -37,7 +40,7 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev`
+`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev curl libcurl4-openssl-dev libssl-dev`
```

Comment:
Thanks! :)
I am trying to build CoreFX on FreeBSD and figuring out many gotchas. I have created many VMs since yesterday for rehearsing and figuring out the bare minimum tools/libs requirements (because the docs are outdated all over the place). Therefore I had kind of have fresh memory buffer on requirements for Roslyn, CoreCLR and CoreFX. (Although none of them are building on FreeBSD at the moment (and i managed to build all of them few weeks ago)).

Thanks for updating the docs! :tada: